### ISSUE TITLE
Invalid config Error

### ISSUE THREAD

**Issue by csmashe** [upsert/lutron-caseta-pro#35]
My install is working fine but i am getting an Invalid config notification.

"The following components and platforms could not be set up: lutron_caseta_pro.scene"

I have no scenes but I thought maybe you would require one so i added once and reran my send integration report. I looked but i do not see my scene in that report.  Everything is working but i would just like the error message to go away.

Below is my report

{
  "LIPIdList" : {
    "Devices" : [
      {
        "ID" : 1,
        "Name" : "Smart Bridge 2",
        "Buttons" : [
          {
            "Name" : "Button 1",
            "Number" : 1
          },
          {
            "Name" : "Button 2",
            "Number" : 2
          },
          {
            "Name" : "Button 3",
            "Number" : 3
          },
          {
            "Name" : "Button 4",
            "Number" : 4
          },
          {
            "Name" : "Button 5",
            "Number" : 5
          },
          {
            "Name" : "Button 6",
            "Number" : 6
          },
          {
            "Name" : "Button 7",
            "Number" : 7
          },
          {
            "Name" : "Button 8",
            "Number" : 8
          },
          {
            "Name" : "Button 9",
            "Number" : 9
          },
          {
            "Name" : "Button 10",
            "Number" : 10
          },
          {
            "Name" : "Button 11",
            "Number" : 11
          },
          {
            "Name" : "Button 12",
            "Number" : 12
          },
          {
            "Name" : "Button 13",
            "Number" : 13
          },
          {
            "Name" : "Button 14",
            "Number" : 14
          },
          {
            "Name" : "Button 15",
            "Number" : 15
          },
          {
            "Name" : "Button 16",
            "Number" : 16
          },
          {
            "Name" : "Button 17",
            "Number" : 17
          },
          {
            "Name" : "Button 18",
            "Number" : 18
          },
          {
            "Name" : "Button 19",
            "Number" : 19
          },
          {
            "Name" : "Button 20",
            "Number" : 20
          },
          {
            "Name" : "Button 21",
            "Number" : 21
          },
          {
            "Name" : "Button 22",
            "Number" : 22
          },
          {
            "Name" : "Button 23",
            "Number" : 23
          },
          {
            "Name" : "Button 24",
            "Number" : 24
          },
          {
            "Name" : "Button 25",
            "Number" : 25
          },
          {
            "Name" : "Button 26",
            "Number" : 26
          },
          {
            "Name" : "Button 27",
            "Number" : 27
          },
          {
            "Name" : "Button 28",
            "Number" : 28
          },
          {
            "Name" : "Button 29",
            "Number" : 29
          },
          {
            "Name" : "Button 30",
            "Number" : 30
          },
          {
            "Name" : "Button 31",
            "Number" : 31
          },
          {
            "Name" : "Button 32",
            "Number" : 32
          },
          {
            "Name" : "Button 33",
            "Number" : 33
          },
          {
            "Name" : "Button 34",
            "Number" : 34
          },
          {
            "Name" : "Button 35",
            "Number" : 35
          },
          {
            "Name" : "Button 36",
            "Number" : 36
          },
          {
            "Name" : "Button 37",
            "Number" : 37
          },
          {
            "Name" : "Button 38",
            "Number" : 38
          },
          {
            "Name" : "Button 39",
            "Number" : 39
          },
          {
            "Name" : "Button 40",
            "Number" : 40
          },
          {
            "Name" : "Button 41",
            "Number" : 41
          },
          {
            "Name" : "Button 42",
            "Number" : 42
          },
          {
            "Name" : "Button 43",
            "Number" : 43
          },
          {
            "Name" : "Button 44",
            "Number" : 44
          },
          {
            "Name" : "Button 45",
            "Number" : 45
          },
          {
            "Name" : "Button 46",
            "Number" : 46
          },
          {
            "Name" : "Button 47",
            "Number" : 47
          },
          {
            "Name" : "Button 48",
            "Number" : 48
          },
          {
            "Name" : "Button 49",
            "Number" : 49
          },
          {
            "Name" : "Button 50",
            "Number" : 50
          },
          {
            "Name" : "Button 51",
            "Number" : 51
          },
          {
            "Name" : "Button 52",
            "Number" : 52
          },
          {
            "Name" : "Button 53",
            "Number" : 53
          },
          {
            "Name" : "Button 54",
            "Number" : 54
          },
          {
            "Name" : "Button 55",
            "Number" : 55
          },
          {
            "Name" : "Button 56",
            "Number" : 56
          },
          {
            "Name" : "Button 57",
            "Number" : 57
          },
          {
            "Name" : "Button 58",
            "Number" : 58
          },
          {
            "Name" : "Button 59",
            "Number" : 59
          },
          {
            "Name" : "Button 60",
            "Number" : 60
          },
          {
            "Name" : "Button 61",
            "Number" : 61
          },
          {
            "Name" : "Button 62",
            "Number" : 62
          },
          {
            "Name" : "Button 63",
            "Number" : 63
          },
          {
            "Name" : "Button 64",
            "Number" : 64
          },
          {
            "Name" : "Button 65",
            "Number" : 65
          },
          {
            "Name" : "Button 66",
            "Number" : 66
          },
          {
            "Name" : "Button 67",
            "Number" : 67
          },
          {
            "Name" : "Button 68",
            "Number" : 68
          },
          {
            "Name" : "Button 69",
            "Number" : 69
          },
          {
            "Name" : "Button 70",
            "Number" : 70
          },
          {
            "Name" : "Button 71",
            "Number" : 71
          },
          {
            "Name" : "Button 72",
            "Number" : 72
          },
          {
            "Name" : "Button 73",
            "Number" : 73
          },
          {
            "Name" : "Button 74",
            "Number" : 74
          },
          {
            "Name" : "Button 75",
            "Number" : 75
          },
          {
            "Name" : "Button 76",
            "Number" : 76
          },
          {
            "Name" : "Button 77",
            "Number" : 77
          },
          {
            "Name" : "Button 78",
            "Number" : 78
          },
          {
            "Name" : "Button 79",
            "Number" : 79
          },
          {
            "Name" : "Button 80",
            "Number" : 80
          },
          {
            "Name" : "Button 81",
            "Number" : 81
          },
          {
            "Name" : "Button 82",
            "Number" : 82
          },
          {
            "Name" : "Button 83",
            "Number" : 83
          },
          {
            "Name" : "Button 84",
            "Number" : 84
          },
          {
            "Name" : "Button 85",
            "Number" : 85
          },
          {
            "Name" : "Button 86",
            "Number" : 86
          },
          {
            "Name" : "Button 87",
            "Number" : 87
          },
          {
            "Name" : "Button 88",
            "Number" : 88
          },
          {
            "Name" : "Button 89",
            "Number" : 89
          },
          {
            "Name" : "Button 90",
            "Number" : 90
          },
          {
            "Name" : "Button 91",
            "Number" : 91
          },
          {
            "Name" : "Button 92",
            "Number" : 92
          },
          {
            "Name" : "Button 93",
            "Number" : 93
          },
          {
            "Name" : "Button 94",
            "Number" : 94
          },
          {
            "Name" : "Button 95",
            "Number" : 95
          },
          {
            "Name" : "Button 96",
            "Number" : 96
          },
          {
            "Name" : "Button 97",
            "Number" : 97
          },
          {
            "Name" : "Button 98",
            "Number" : 98
          },
          {
            "Name" : "Button 99",
            "Number" : 99
          },
          {
            "Name" : "Button 100",
            "Number" : 100
          }
        ]
      },
      {
        "ID" : 13,
        "Name" : "Unnamed Pico Remote 1",
        "Buttons" : [
          {
            "Number" : 2
          },
          {
            "Number" : 3
          },
          {
            "Number" : 4
          },
          {
            "Number" : 5
          },
          {
            "Number" : 6
          }
        ]
      },
      {
        "ID" : 12,
        "Name" : "Unnamed Pico Remote 2",
        "Buttons" : [
          {
            "Number" : 2
          },
          {
            "Number" : 3
          },
          {
            "Number" : 4
          },
          {
            "Number" : 5
          },
          {
            "Number" : 6
          }
        ]
      }
    ],
    "Zones" : [
      {
        "ID" : 2,
        "Name" : "Ceiling Fan",
        "Area" : {
          "Name" : "Office"
        }
      },
      {
        "ID" : 3,
        "Name" : "Ceiling Fan Light",
        "Area" : {
          "Name" : "Office"
        }
      },
      {
        "ID" : 4,
        "Name" : "Main Lights",
        "Area" : {
          "Name" : "Front Foyer"
        }
      },
      {
        "ID" : 5,
        "Name" : "Front Lights",
        "Area" : {
          "Name" : "Outside"
        }
      },
      {
        "ID" : 6,
        "Name" : "Overhead Lights",
        "Area" : {
          "Name" : "Kitchen"
        }
      },
      {
        "ID" : 7,
        "Name" : "Sink Light",
        "Area" : {
          "Name" : "Kitchen"
        }
      },
      {
        "ID" : 8,
        "Name" : "Back Porch Lights",
        "Area" : {
          "Name" : "Outside"
        }
      },
      {
        "ID" : 9,
        "Name" : "Side Lights",
        "Area" : {
          "Name" : "Family Room"
        }
      },
      {
        "ID" : 10,
        "Name" : "Ceiling Fan",
        "Area" : {
          "Name" : "Family Room"
        }
      },
      {
        "ID" : 11,
        "Name" : "Ceiling Fan Light",
        "Area" : {
          "Name" : "Family Room"
        }
      }
    ]
  }
}

**Comment by upsert** [comment]
I tried your integration report and didn't see any errors. Tested on 0.95.4 with the current custom component. Are you using Home Assistant 0.95.4 and the latest custom component?

Try temporarily adding this to your `configuration.yaml` to log more details:
```yaml
logger:
  default: debug
```

If HA is producing errors, there should be more details in the logs.

For your scenes, they would normally be where it says "Button 1", "Button 2" etc. in your integration report.
